### PR TITLE
Chore/readme cleanup

### PR DESCRIPTION
**Pull request by ffriedl89**

### CODE REVIEW

**Comment by thomaspink** [comment]
LGTM

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -105,24 +105,3 @@ To test the rules run
 ```
 npm run tslint:test
```
Comment (lukasholzer) [inline DEVELOPMENT.md]: this command does not exist and work anymore

Comment (lara-aigmueller) [reply DEVELOPMENT.md]: As you are the one currently knowing all/most of the commands, could you at some point update the documentation files accordingly?

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -92,37 +86,8 @@ npm run demos
...
-./gradlew lint
-./gradlew compile
-```
-
-To see complete list of gradlew builds, run:
+If you want to run the design system itself including the demos embedded use
 
 ```
-./gradlew tasks
+npm run barista
```
Comment (tomheller) [inline DEVELOPMENT.md]: barista task does not exist in the package.json scripts

Comment (thomaspink) [reply DEVELOPMENT.md]: it does ;) https://github.com/dynatrace-oss/barista/blob/master/package.json#L33

**Pull request merged**